### ISSUE TITLE
Keyboard shortcut overwrites are not working in v1.75.0

### ISSUE THREAD

**Issue by KarelJanVanHaute** [microsoft/vscode#173801]
Type: <b>Bug</b>

Since the update, keyboard shortcuts overwrites are not longer working. 
I have this in my keybindings.json file. 

```
  {
    "key": "cmd+[Period]",
    "command": "-breadcrumbs.focus",
    "when": "breadcrumbsPossible && breadcrumbsVisible"
  },
  {
    "key": "shift+cmd+[Period]",
    "command": "breadcrumbs.focus",
    "when": "breadcrumbsPossible && breadcrumbsVisible"
  },
  {
    "key": "shift+cmd+[Period]",
    "command": "-editor.action.commentLine",
    "when": "editorTextFocus && !editorReadonly"
  },
  {
    "key": "cmd+[Period]",
    "command": "editor.action.commentLine",
    "when": "editorTextFocus && !editorReadonly"
  }
```

VS Code version: Code 1.75.0 (e2816fe719a4026ffa1ee0189dc89bdfdbafb164, 2023-02-01T15:26:31.005Z)
OS version: Darwin arm64 22.2.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|6, 7, 8|
|Memory (System)|16.00GB (0.05GB free)|
|Process Argv|--crash-reporter-id c8ecf15c-9991-465a-88ea-197a5adccd07|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (51)</summary>

Extension|Author (truncated)|Version
---|---|---
project-manager|ale|12.7.0
gitflow-actions-sidebar|ard|0.5.2
vscode-mjml|att|1.6.0
vscode-tailwindcss|bra|0.9.7
path-intellisense|chr|2.8.4
laravel-blade|cjh|1.1.2
vscode-eslint|dba|2.4.0
javascript-ejs-support|Dig|1.3.1
xml|Dot|2.5.1
Vue2|duc|0.1.3
gitlens|eam|13.2.0
prettier-vscode|esb|9.10.4
vscode-todo-plus|fab|4.19.1
auto-close-tag|for|0.5.14
auto-rename-tag|for|0.1.10
copilot|Git|1.71.8269
copilot-labs|Git|0.9.668
headwind|hey|1.7.0
minify|Hoo|0.4.3
vscode-htmlhint|HTM|1.0.4
rem-rpx|ica|1.0.4
snipped|jef|1.3.0
svg|joc|1.5.0
vscode-peacock|joh|4.2.2
pretty-formatter|mbl|0.2.2
twig-language-2|mbl|0.9.2
template-string-converter|meg|0.6.0
dotenv|mik|1.0.1
ecdc|mit|1.8.0
vscode-filesize|mkx|3.1.0
vscode-apache|mrm|1.2.0
vscode-postgresql|ms-|0.3.0
vsliveshare|ms-|1.0.5828
color-highlight|nau|2.5.0
vetur|oct|0.36.1
laravel-blade|one|1.32.0
vscode-versionlens|pfl|1.3.0
prisma|Pri|4.10.0
vscode-commons|red|0.0.6
vscode-yaml|red|1.11.0
vue-vscode-snippets|sdr|3.1.1
code-spell-checker|str|2.16.0
code-spell-checker-dutch|str|0.3.7
es6-string-html|Tob|2.12.0
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
fig|wit|0.0.7
vscode-import-cost|wix|3.3.0
html-css-class-completion|Zig|1.20.0
vue|znc|0.12.0
vue-language-features|znc|0.12.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
pyindex848cf:30577861
nodejswelcome1cf:30587006
2e4cg342:30602488
89544117:30613380
pythonsymbol12:30657548
2i9eh265:30646982

```

</details>

**Comment by gjsjohnmurray** [comment]
Probably https://github.com/microsoft/vscode/issues/173325 which is fixed in Insiders and in upcoming 1.75.1

**Comment by alexdima** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/173325
